### DISCUSSION TITLE
What is the realistic limit to using a Raspberry Pi for actual mining?

### DISCUSSION THREAD

**Discussion by number435398** [Chia-Network/chia-blockchain#5011] [Support]
A single USB controller can handle up to 127 devices (including hubs). So, a single raspberry pi could, theoretically, be connected to over 100 of the 32tb external hard drives, which would work out to 3.2pb.  Or 1pb from 100 of the 10tb drives. Can a single raspberry pi really go through that much data in a timely manner so as to truly mine Chia? I feel like there's either a dynamic I'm unaware of to the mining of this coin, or there's got to be an upper limit that's not being discussed.  Some external drives are slow to begin with and combined with their space, all being searched by something as computationally pathetic (in the crypto world) as a raspberry pi, I am truly wondering what the ultimate truth is.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
